test/test_models.py

```diff
@@ -148,10 +145,9 @@ def _test_detection_model(self, name, dev):
         set_rng_seed(0)
         kwargs = {}
         if "retinanet" in name:
-            kwargs["score_thresh"] = 0.013
+            # Reduce the default threshold to ensure the returned boxes are not empty.
+            kwargs["score_thresh"] = 0.01
```

Comment:
Use a threshold that will produce many more boxes and make things more interesting.